plugins/Kaleidoscope-Ranges/src/Kaleidoscope-Ranges.h

```diff
@@ -87,6 +88,8 @@ enum : uint16_t {
   OS_CANCEL,
   CS_FIRST,
   CS_LAST = CS_FIRST + MAX_CS_KEYS,
+  QK_FIRST,
+  QK_LAST = QK_FIRST + MAX_QK_KEYS,
```

Comment:
Asked a few questions about this on discord, so might change this depending on answers there